oxidation/libparsec/crates/core_fs/src/error.rs

```diff
@@ -6,10 +6,9 @@ use thiserror::Error;
 use uuid::Uuid;
 
 use libparsec_crypto::CryptoError;
-use libparsec_platform_local_db::DatabaseError;
 use libparsec_types::{EntryID, FileDescriptor};
 
-#[derive(Error, Debug, PartialEq, Eq)]
+#[derive(Error, Debug, PartialEq)]
```

Comment:
Why ?

Reply:
That's a miss, previously I had added the variant that store the `StorageError` which don't implement `Eq` trait. 

But now can revert that